install_lichen.sh

```diff
@@ -54,8 +54,9 @@ fi
 ####################################################################################################
 # compile & install the hash comparison tool
 
-pushd "${lichen_repository_dir}" > /dev/null
-clang++ -I "${lichen_vendor_dir}" -lboost_system -lboost_filesystem -Wall -Wextra -Werror -g -Ofast -flto -funroll-loops -std=c++11 compare_hashes/compare_hashes.cpp compare_hashes/submission.cpp -o "${lichen_installation_dir}/compare_hashes/compare_hashes.out"
+pushd "${lichen_installation_dir}/compare_hashes" > /dev/null
+cmake "$lichen_repository_dir/compare_hashes"
```

Comment:
I'm not sure what cmake will default to but on ubuntu based OS's I would think its `gcc` you may want to set in the makefile or in the `CXX` env variable to force use clang if its preferred to keep the same compiler

Reply:
There's no real preference here.  I think the original reason I chose clang was simply because I couldn't get gcc to work due to some sort of flag ordering thing.